src/WpfMath/Atoms/RowAtom.cs

```diff
@@ -46,24 +46,26 @@ public RowAtom(SourceSpan source, Atom baseAtom)
         public RowAtom(SourceSpan source)
             : base(source)
         {
-            this.Elements = new List<Atom>().AsReadOnly();
+            this.Children = new List<Atom>().AsReadOnly();
         }
 
         private RowAtom(SourceSpan source, DummyAtom previousAtom, ReadOnlyCollection<Atom> elements)
             : base(source)
         {
             this.PreviousAtom = previousAtom;
-            this.Elements = elements;
+            this.Children = elements;
         }
 
         internal RowAtom(SourceSpan source, IEnumerable<Atom> elements)
-            : base(source) =>
-            this.Elements = elements.Where(x => x != null).ToList().AsReadOnly();
+            : base(source)
+        {
+            this.Children = elements.Where(x => x != null).ToList().AsReadOnly(); ;
+        }
             // TODO[F]: Fix this with C# 8 migration: there shouldn't be nullable atoms in this collection
 
         public DummyAtom PreviousAtom { get; }
 
-        public ReadOnlyCollection<Atom> Elements { get; }
+        public ReadOnlyCollection<Atom> Elements => (ReadOnlyCollection<Atom>)this.Children;
```

Comment:
I don't like this cast. Let's make it the other way around? Leave poor `Elements` as-is and implement the new `Children` property via `Elements`:
```
public override IEnumerable<Atom> Children => Elements;
```